### PR TITLE
Symfonize admin index

### PR DESCRIPTION
**Pull request by alexarn**

### CODE REVIEW

File: templates/admin/index.html.twig [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+    {% if access_model %}
+      <li><a href='{{ asset("model") }}'>Les modèles</a></li>
+    {% endif %}
+
+    {% if userCan('22') %}
+      <li><a href='{{ asset("framework") }}'>Les tableaux</a></li>
+    {% endif %}
+
+    {% if userCan('1101') %}
+      <li><a href='{{ asset("closingday") }}'>Jours de fermeture</a></li>
```
Comment (jeromecombes) [inline templates/admin/index.html.twig:31]: For closingdays :
<pre>{% if userCan('25') %}</pre>

**Review by jeromecombes — CHANGES_REQUESTED** [review]

File: templates/admin/index.html.twig [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+    {% endif %}
+
+    {% if userCan('1101') and config('PlanningHebdo') %}
```
Comment (jeromecombes) [inline templates/admin/index.html.twig:34]: userCan('1101') does not work.
Do what you've done for models

**Review by jeromecombes — CHANGES_REQUESTED** [review]

File: templates/admin/index.html.twig (hunk 2 above)
Comment (jeromecombes) [reply templates/admin/index.html.twig]: fixed with 9c39534d0ddb84423da051cd17ebe0425e27eedc

File: templates/admin/index.html.twig (hunk 1 above)
Comment (jeromecombes) [reply templates/admin/index.html.twig:31]: fixed with 9c39534

**Review by jeromecombes — APPROVED** [review]

**Comment by alexarn** [comment]
Merged on master

**Pull request closed without merging**